git/githistory/log/log.go

```diff
@@ -225,3 +229,10 @@ func (l *Logger) logLine(str string) (n int, err error) {
 func (l *Logger) log(str string) (n int, err error) {
 	return fmt.Fprint(l.sink, str)
 }
+
+func maxInt(a, b int) int {
+	if a > b {
+		return a
+	}
+	return b
+}
```

Comment:
Why'd you pull this out? Is there a circular dependency now?

Reply:
Great question, this happened when importing the `git/githistory/log` package from `progress`, since both of those packages import `tools`. This is the only reference to the `tools` package inside of `git/githistory/log`, so this was sufficient to remove the cyclic dependency.